cuenca/resources/kyc_validations.py

```
@@ -24,18 +25,19 @@ class Config:
                 'attemps': '1',
             }
         }
+    )
 
     @classmethod
     def create(
         cls,
         user_id: str,
         force: bool = False,
-        documents: List[KYCFile] = [],
+        documents: list[KYCFile] = [],
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)